spec/lib/atom_serializer_spec.rb

```diff
@@ -190,8 +190,24 @@
   end
 
   describe '#follow_request_salmon' do
-    it 'returns dumpable XML'
-    it 'triggers follow request when processed'
+    let(:follow_request) { Fabricate(:follow_request, account: author, target_account: receiver) }
+    let(:atom_serializer) { described_class.new }
+    subject { atom_serializer.follow_request_salmon(follow_request) }
+
+    it { is_expected.to be_instance_of Ox::Element }
```

Comment:
Actually, most of `AtomSerializer`'s method returns `Ox::Element` . I think we should test what included by the `Ox::Element`.